settings.gradle

```diff
@@ -52,7 +52,7 @@ include(':ballerina-streaming')
 include(':ballerina-logging')
 include(':ballerina-launcher')
 include(':ballerina-packerina')
-include(':ballerina-backend-llvm')
+//include(':ballerina-backend-llvm')
```

Comment:
Shall we remove these dependencies if not needed?

Reply:
Commented because this module is commented out in build profiles in the parent pom as well. I think it's just temporarily disabled. So just commented it out in Gradle files as well.